src/vivarium_csu_alzheimers/data/loader.py

```diff
@@ -78,16 +79,30 @@ def load_population_location(
     return location
 
 
-def load_population_structure(
+def load_forecast(param: str, location: str, years: int | str | list[int]) -> pd.DataFrame:
+    loc_id = utility_data.get_location_id(location)
+    age_mapping = get_data(data_keys.POPULATION.AGE_BINS, location, years)
+    return table_from_nc(FORECAST_NC_FNAME_DICT, "population", loc_id, location, age_mapping)
+
+
+def load_forecasted_population_structure(
+    key: str, location: str, years: int | str | list[int] | None = None
+) -> pd.DataFrame:
+    population_df = load_forecast("population", location, years)
+    population_df["value"] = population_df.mean(axis=1)
+    return population_df.filter(like="value")
+
+
+def load_forecasted_mortality(
     key: str, location: str, years: int | str | list[int] | None = None
 ) -> pd.DataFrame:
-    return interface.get_population_structure(location, years)
+    return load_forecast("mortality", location, years).loc[location]
```

Comment:
Why do we have `.loc[location]`? Isn't the result of `load_forecast()` a wide DataFrame with draw columns, which is already filtered to the location we want?

Reply:
+ 1 this. Also isn't consistent with the call in load_forecasted_population_structure.